plugins.audio.pbsfm/addon.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.audio.pbsfm" name="PBS FM" version="1.0.0" provider-name="Andras Nemes">
+    <requires>
+        <import addon="xbmc.python" version="2.25.0"/>
+        <import addon="script.module.beautifulsoup" version="3.0.8"/>
+		<import addon="script.module.kodiswift" version="0.0.7"/>
+    </requires>
+    <extension point="xbmc.python.pluginsource" library="addon.py">
+        <provides>audio</provides>
+    </extension>
+    <extension point="xbmc.addon.metadata">
+    <platform>all</platform>
+    <language></language>
+    <summary lang="en">Stream and ondemand player for Melbourne music station PBSfm.</summary>
+    <description lang="en">Since 1979, community broadcaster PBS 106.7FM has been an integral player in Melbourne's diverse music community - with more than 80 specialist music programs ranging from soul to garage to country to jazz - PBS is dedicated to nurturing, inspiring and championing Melbourne's diverse music community. 
+    </description>
+    <license>The MIT License (MIT)</license>
+    <email>andras.nemes@gmail.com</email>
+    <forum></forum>
```

Comment:
please consider to add the assets tag in a new update: http://kodi.wiki/view/addon.xml